impl.go

```diff
@@ -98,13 +103,22 @@ func stripPaths(in string) string {
 			break
 		}
 		runes = runes[n:]
+		if isQuotedPath && len(runes) > 0 {
+			runes = runes[1:]
+		}
 
 		// Copy non-path runes verbatim
 		n = slices.IndexFunc(runes, isPathRune)
 		seg = runes
 		if n >= 0 {
 			seg = seg[:n]
 		}
+		if n > 0 && seg[0] == '"' {
```

Comment:
Please add hasPrefix and hasSuffix helpers instead of checking n and n[0]. (Or maybe even better would be trimPrefix and trimSuffix? On my phone.)